wp-content/themes/pub/wporg-learn-2024/inc/block-config.php

```diff
@@ -534,7 +534,7 @@ function get_student_course_options( $options ) {
 	$label = $options[ $selected_slug ] ?? $options['all'];
 
 	return array(
-		'label' => $label,
+		'label' => sprintf( __( 'Status: %s', 'wporg-learn' ), $label ),
 		'title' => __( 'Completion status', 'wporg-learn' ),
```

Comment:
Can you change it to upper case for 'status' to make it consistent?

Reply:
Yes, let's use sentence case. Thanks for checking!